egs/librispeech/ASR/conformer_ctc/transformer.py

```diff
@@ -179,10 +180,15 @@ def forward(
               memory_key_padding_mask for the decoder. Its shape is (N, T).
               It is None if `supervision` is None.
         """
-        if self.use_feat_batchnorm:
+        if (
+            isinstance(self.use_feat_batchnorm, bool)
+            and self.use_feat_batchnorm
+        ):
             x = x.permute(0, 2, 1)  # (N, T, C) -> (N, C, T)
             x = self.feat_batchnorm(x)
             x = x.permute(0, 2, 1)  # (N, C, T) -> (N, T, C)
+        if isinstance(self.use_feat_batchnorm, float):
+            x *= 0.1
```

Comment:
I think you wanted 

x *= self.use_feat_batchnorm

Reply:
Ah, yes.  Thanks.